recipes/pygmi/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "pygmi" %}
+{% set version = "3.2.9.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pygmi-{{ version }}.tar.gz
+  sha256: 9701aa93368d569cb971eb50eed0524a7cac2bf9562ce7404b1ade7d9e508490
+
+build:
+  noarch: python
+  entry_points:
+    - pygmi-cli = pygmi.main:main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python 3.10
```

Comment:
Replace explicitly specifying `3.10` here and in the sections below, by setting a jinja variable for `python_min`. See:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python